cloudigrade/util/aws/sqs.py

```diff
@@ -39,6 +39,9 @@ def delete_message_from_queue(queue_url, messages):
         dict: The response from the delete call.
 
     """
+    if not bool(messages):
```

Comment:
In addition to adding a test for this, also adding an info/debug log would be helpful.

Reply:
This happens every time the analyzer runs if instances haven't changed.  I can add this but it will be common.

Reply:
Sounds like a debug log would be appropriate then.